code/game/machinery/vendors/generic_vendors.dm

```diff
@@ -687,6 +687,10 @@
 					/obj/item/clothing/under/misc/pinkhawaiianshirt = 1,
 					/obj/item/clothing/under/misc/bluehawaiianshirt = 1,
 					/obj/item/clothing/under/misc/orangehawaiianshirt = 1,
+					/obj/item/clothing/suit/jacket/varsity = 1,
+					/obj/item/clothing/suit/jacket/varsity_sport = 1,
+					/obj/item/clothing/suit/jacket/varsity_blood = 1,
+					/obj/item/clothing/suit/jacket/driver = 1,
```

Comment:
Do you consider them to be costumes? If not then clothesmate would be the place.

Reply:
yeah i thought its not exactly regular clothes